src/hott/05-sigma.rzk.md

```diff
@@ -6,16 +6,21 @@ This is a literate `rzk` file:
 #lang rzk-1
 ```
 
-It is convenient to have a shorthand for `Σ (x : A), C x`
-which avoids explicit naming the variable `x : A`.
+It is convenient to have a shorthand for `Σ (x : A), B x` which avoids explicit
+naming the variable `x : A`.
 
 ```rzk
 #def total-type
   ( A : U)
-  ( C : A → U)
+  ( B : A → U)
   : U
-  :=
-    Σ (x : A), C x
+  := Σ (x : A), B x
+
+#def total-type-projection
```

Comment:
`projection-total-type`, since the conclusion is the projection, not the total type.